### PR TITLE
Add ext config

### PR DESCRIPTION
**Pull request by pavlovic-ivan**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const { createOrDelete } = require("../infra")
 const uuid = require("uuid");
-
-const jobFilter = process.env.JOB_FILTER;
+const fs = require('fs');
+const { config } = require("process");
```
Comment (jgiannuzzi) [inline src/app.js:4]: I don't think you use this

File: .github/workflows/aws_sam.yml [hunk 2]
```diff
@@ -9,8 +9,14 @@ jobs:
...
     name: Test
     runs-on: ubuntu-latest
+    environment: protected
     steps:
       - uses: actions/checkout@v3
+
+      - name: Transform YAML to JSON and echo to file
+        run: |
+          echo "${{ vars.CONFIG }}" | yq e -j - > ./config.json
+          ls -la
```
Comment (jgiannuzzi) [inline .github/workflows/aws_sam.yml:19]: probably best to keep the config in json all along, or to document the yaml format in an example too

File: src/app.js (hunk 1 above)
Comment (pavlovic-ivan) [reply src/app.js]: removed

File: .github/workflows/aws_sam.yml (hunk 2 above)
Comment (pavlovic-ivan) [reply .github/workflows/aws_sam.yml]: switched to yaml throughout the project

**Pull request merged**